docs/docs/user-guide/permissions.md

```diff
@@ -0,0 +1,108 @@
+
+- **Password Management**: For each role, you can configure or change the password directly within Mathesar.
+- **Remove Role Configuration**: You can remove role configurations from Mathesar as needed. This only removes the credentials, not the role from the server.
+
+### Collaborators
+
+Collaborators are users who have been added to work on a database. Each collaborator is assigned a PostgreSQL LOGIN role that dictates their level of access to the database and its objects (schemas, tables, etc.).
+
+- **Add Collaborators**: In the **Collaborators** section, you can add new users as collaborators to the database. When adding a collaborator, you assign them one of the roles that have been configured in Mathesar.
+
+- **Remove Collaborators**: You can remove collaborators if they no longer require access to the database. The Mathesar user will still exist and you can add them back as a collaborator at a later time.
+
+### Server Roles
+
+The **Server Roles** section, found under the **Settings** tab, shows the roles available on the server for the current database.
+
+These roles will be displayed for all databases on the server. Also, any changes made to a server role will be reflected in all databases on the server.
+
+- **Create Roles**: You can create new server-level roles from this section. You can configure these roles in two ways:
+    1. With login capability and a password, which you can assign to collaborators.
+    2. Without login capability, to be used as a parent role exclusively. You cannot assign these roles to collaborators directly.
+- **Define Child Roles**: PostgreSQL has a mechanism for [Role Membership](https://www.postgresql.org/docs/current/role-membership.html) wherein any role can be "granted" to any other role to form simple hierarchies or complex graph-based inheritance structures. For any role you've configured within Mathesar, you can use Mathesar to grant the role to other "child roles".
+- **Drop Roles**: You can drop server-level roles that are no longer needed. This action removes the role from the server and all databases where it has been assigned. Exercise caution when dropping roles, as it may affect existing permissions and user access across multiple databases.
+
+!!! note
+    Server roles, once added, must be configured in Mathesar under the **Role Configuration** section before they can be assigned to collaborators.
+
+---
+
+## Permissions Management at Different Levels
+
+Mathesar manages permissions at three levels:
+
+1. **Database Level:** Managed from the database page.
+2. **Schema Level:** Managed from the schema page.
+3. **Table Level:** Managed from the inspector panel for each table.
+
+### Ownership of Objects
+
+Owners of each object (database, schema, or table) can set permissions for other users. Ownership is determined by the user's assigned collaborator role in the database.
```

Comment:
> Ownership is determined by the user's assigned collaborator role in the database

This statement is a bit confusing. It might be better to not include this line since we explain it below.

Reply:
I have removed this line.